react-debugging.html

```diff
@@ -11,13 +11,16 @@
     <title>Mood Meter</title>
   </head>
   <body>
-    <div id="react-container"></div>
+    <div id="react-container">I couldn't get it to render the React code :(</div>
     <script type="text/babel">
     // No need to change anything ABOVE this line ^
+    
+    //Fix #1: add import
+    import React, {Component} from 'react' 
+  
 
-
-
-    class App extends React.Component{
+    //Fix #2: fix syntax of the extends
+    class App extends Component {
```

Comment:
The original class extends react.component was actually correct, again, mostly due to the fact this is all built into the HTML page thus the syntax is different from a full blown standalone react app.